PoGo.NecroBot.Logic/Model/Settings/SnipeFIlter.cs

```diff
@@ -145,7 +145,7 @@ public IPokemonFilter GetGlobalFilter()
             return new SnipeFilter(_setting.MinIVForAutoSnipe)
             {
                 SnipeIV = _setting.MinIVForAutoSnipe, 
-                Operator = "and",
+                Operator = "or", //default and is not good becouse you don't have default skills so in IsMatch you will always have false
                 Priority = 5,
```

Comment:
but this operator is not between lvl and iv but between iv and skills. Default there are no skills so is always false.
lvl and iv is always by and